Documentation/instruction_name_tables.md

```diff
@@ -0,0 +1,62 @@
+## Database table naming rules
+
+Before uploading a new dataset in the database and creating a new table, the following rules should be followed to name the table:
+
+### All names:
+1. Always in lower case; 
+2. No space between the characters. Instead of using space, use `_` to separate composite names.
+3. Column names should follow the above rules.
+
+
+### Schema names:
+
+If the table it's single country data use the full ISO country name. This name for each country of the world can be found [here](https://gadm.org/maps.html).
+
+If the data is on a global scale data use the schema: `global`.
+
+
+### Table names:
+
+Table names should follow this basic structure:
+
+`<locality>_<semantic single word name>_<0|D|W|M>_<optional suffix>`
+
+The symbols <> are just delimiters. On PostGIS tables representing maps, a `gis` prefix should be added. 
+
+The `<locality>` will be filled if the data in the table belongs to a specific administrative region of the country. In this case, you should name the locality according to the nomenclature adopted in [GADM](https://gadm.org/maps.html). You can omit this term if the data refers to the entire region that named the schema, e.g., the whole country.
+
+The `<semantic single word name>` means that the words chosen to represent the data frame must express the content of the table and not be a simple abbreviation of the content. 
+
+The `<0|D|W|M>` portion informs about the periodicity of the update. It should use D for daily, W for weekly, M for monthly, and 0 for static.
```

Comment:
`<0|D|W|M>` probably, it should be in lowercase

Reply:
Agreed with the lowercase to maintain consistency with what was defined above.